opencti-worker/Dockerfile

```
@@ -1,23 +1,14 @@
-# Use the official Docker Hub Ubuntu 18.04 base image
-FROM ubuntu:19.04
-
-# Update the base image
-RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get -y upgrade && apt-get -y dist-upgrade
-
-# Setup install environment and OpenCTI dependencies
-RUN DEBIAN_FRONTEND=noninteractive apt-get -y install python3 python3-pip curl rsyslog wget netcat locales
-
-# Set the locale
-RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && locale-gen
-ENV LANG en_US.UTF-8  
-ENV LANGUAGE en_US:en  
-ENV LC_ALL en_US.UTF-8 
+FROM python:3.7.4-alpine3.10
 
 # Copy the worker
 COPY src /opt/opencti-worker
 
 # Install Python modules
-RUN cd /opt/opencti-worker && pip3 install -r requirements.txt
+RUN BUILD_WORKER=noninteractive \
+    apk add build-base && \
```

`--no-cache` -> https://github.com/hadolint/hadolint/wiki/DL3019
version pinning -> https://github.com/hadolint/hadolint/wiki/DL3018
```suggestion
    apk add --no-cache build-base=0.5-r1 && \
```

---

Thanks for your comment. I will take some times to dig into hadolint and fix that.